client/components/editor.vue

```diff
@@ -121,6 +145,16 @@
         fixToolbarToTop: false,
         originalToolbarYPos: -1,
         toolbarWidth: {width: '100%'},
+        dispatch: null,
+        state: null,
+        link: '',
+        linkDialog: false,
+        selectedExistingLinks: [],
+        validLink: false,
+        linkValidationRule: (value) => {
+          const urlRegex = /^(https?:\/\/)?((([a-z\d]([a-z\d-]*[a-z\d])*)\.)+[a-z]{2,}|((\d{1,3}\.){3}\d{1,3}))(:\d+)?(\/[-a-z\d%_.~+]*)*(\?[;&a-z\d%_.~+=-]*)?(#[-a-z\d_]*)?$/i;
```

Comment:
This prevents one to entry `mailto` URLs.

Let's just have a simple general URI check, not requiring HTTP.

Reply:
I'll change the regex to accept any scheme (http, mailto, etc.)